libs/sdk-backend-spi/src/workspace/execution/export.ts

```diff
@@ -23,20 +23,30 @@ export interface IExportConfig {
     mergeHeaders?: boolean;
 
     /**
-     * Applicable for XLSX format; specifies filters to include as comments / metadata in
+     * Applicable for XLSX, and PDF format; specifies filters to include as comments / metadata in
      * the Excel sheet.
      *
      * @remarks
      * Filters provided here are purely to paint a better context for the
-     * person looking at the XLSX file. They serve no other purpose and are merely serialized
-     * into the XLSX in a human readable form.
+     * person looking at the exported file. They serve no other purpose and are merely serialized
+     * in the export in a human-readable form.
      */
     showFilters?: boolean;
```

Comment:
@dremileta please add the remark that for PDF, the `visualizationObjectId` has to be provided to make it work in the PDF format, otherwise, other devs  using it may fall into the same issue as we did